### PR TITLE
Prepare readme for the 1.0.0 release.

### LINKED ISSUE #37: Work on readme before making 1.0 release
Along with the migration to AiiDA 1.0 the README file needs to be adapted to reflect the changes.

### PR DESCRIPTION
**Pull request by yakutovicha**
fixes #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
 [![PyPI version](https://badge.fury.io/py/aiida-cp2k.svg)](https://badge.fury.io/py/aiida-cp2k)
 
 # AiiDA CP2K
-**The CP2K plugin for the AiiDA workflow and provenance engine.**
+[AiiDA](http://www.aiida.net/) plugin for [CP2K](https://www.cp2k.org/).
 
-
-![logo](./aiida-cp2k_logo.png)
+# Documentation
```
Comment (ltalirz) [inline README.md]: probably use ## everywhere except for the top one

File: README.md [hunk 2]
```diff
@@ -15,88 +15,36 @@ If you use `pip`, you can install it as:
 pip install aiida-cp2k
 ```
 
-
-# Features
```
Comment (ltalirz) [inline README.md]: I really like such lists of features in the readme - I believe it is typically a sign of good design if what a python package does can be explained succinctly in a few words, and I would suggest to keep it here.

One alternative could be to keep the README absolutely minimal (nothing more than a pointer to the docs) and move everything to the docs. Up to you, but whatever you decide, I would suggest to clearly define for yourself what type of content should go in the README and what in the docs.

File: README.md (hunk 1 above)
Comment (yakutovicha) [reply README.md]: Yep, I agree that it looks better.

File: README.md (hunk 2 above)
Comment (yakutovicha) [reply README.md]: I will keep it minimal. I think this way is much better.

File: README.md [hunk 3]
```diff
@@ -3,100 +3,36 @@
 [![PyPI version](https://badge.fury.io/py/aiida-cp2k.svg)](https://badge.fury.io/py/aiida-cp2k)
```
Comment (ltalirz) [inline README.md]: how about adding an rtd badge here?

File: README.md [hunk 4]
```diff
@@ -3,100 +3,36 @@
...
 ```
 
-- Section parameters are stored as key `_` in the dictionary:
-```
-xc_section = {'XC_FUNCTIONAL': {'_': 'LDA'}}
-```
+## Links
+* [Documentation](https://aiida-cp2k.readthedocs.io/en/latest/) for the calculation examples and features of the plugin.
+* [Make an issue](https://github.com/aiidateam/aiida-cp2k/issues/new) for bug reports, questions and suggestions.
+* [AiiDA](http://www.aiida.net/) to learn about AiiDA.
```
Comment (ltalirz) [inline README.md]: aiida & cp2k you could already link in the text directly

File: README.md (hunk 3 above)
Comment (yakutovicha) [reply README.md]: thanks, added it.

File: README.md (hunk 4 above)
Comment (yakutovicha) [reply README.md]: ok

**Pull request merged**